portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,149 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+}
+
+body{
+   background-color: #f3f5f9;
+}
+
+.wrapper{
+  display: flex;
+  position: relative;
+}
+
+/* -------------- Side Bar --------------*/
+
+.wrapper .sidebar{
+  width: 280px;
+  height: 100%;
+  background: #6767b5;
+  padding: 30px 0px;
+  position: fixed;
+}
+
+/* -------------- Side Bar Avatar --------------*/
+
+.wrapper .sidebar .avatar {
+    position: relative;
+    overflow: hidden;
+    margin: 55px;
+    width: 170px;
+    height: 170px;
```

Comment:
It's usually better to put width/height as attributes on an image instead of CSS. This way while the image loads, the browser knows how much room the image is going to take up so less pixels are shifted once the image fully loads.